tests/EdFi.Ods.Api.IntegrationTests/EducationOrganizationTestDataBuilder.cs

```diff
@@ -262,6 +258,33 @@ public EducationOrganizationTestDataBuilder AddCommunityProvider(int communityPr
             return this;
         }
 
+        public EducationOrganizationTestDataBuilder AddParent(string newGuidId)
+        {
+            _sql.AppendLine(
+                $@"INSERT INTO edfi.Parent (
+                    FirstName,
+                    LastSurname,
+                    ParentUniqueId)
+                VALUES(
+                    '{newGuidId}',
+                    '{newGuidId}',
+                    '{newGuidId}');"
+            );
+
+            return this;
+        }
+
+
+
+        public EducationOrganizationTestDataBuilder AddStudentParentAssociation(int parentUSI, int studentUSI)
+        {
+            _sql.AppendLine(
+                 $@"INSERT INTO edfi.StudentParentAssociation (ParentUSI,StudentUSI)
+                VALUES (" + parentUSI + "," + studentUSI + ");");
```

Comment:
Why suddenly go to string concatenation when you're using interpolation at the beginning?

```suggestion
                VALUES ({parentUSI}, {studentUSI});");
```